tidy3d/plugins/mode/solver.py

```diff
@@ -1011,6 +1012,26 @@ def split_curl_field_postprocess_inverse(split_curl, E):
         """E has the shape (3, N, num_modes)"""
         raise RuntimeError("Split curl not yet implemented for relative mode solver.")
 
+    @staticmethod
+    def mode_plane_contain_good_metal(material_response) -> bool:
+        """Find out if epsilon on the modal plane contain very metallic materials whose permittivity
+        or permeability value is very large.
```

Comment:
I think we can refer to these materials as "good" conductors, so it might help to use the same wording here too.

Reply:
I'm not sure what the correct terminology here, but I feel "good" sounds a bit strange overall here? maybe "strong" conductors or something. again not super set on this but just a feeling I had. basically it feels a little vague to me